src/cryptonote_core/cryptonote_core.cpp

```diff
@@ -691,6 +697,10 @@ namespace cryptonote
     if (block_sync_size > BLOCKS_SYNCHRONIZING_MAX_COUNT)
       MERROR("Error --block-sync-size cannot be greater than " << BLOCKS_SYNCHRONIZING_MAX_COUNT);
 
+    blocks_batch_max_size = command_line::get_arg(vm, arg_blocks_batch_max_size);
+    if (blocks_batch_max_size > BLOCKS_BATCH_MAX_SIZE)
+      MERROR("Error --block-batch-max-size cannot be greater than " << BLOCKS_BATCH_MAX_SIZE);
```

Comment:
20mb is a sane default, but a user should be able to increase it if their hardware allows for it.

I think we can allow the custom value to be set as high as 50mb (1/2 of the packet size limit), perhaps even 100mb to allow a user access to the full packet size limit